data-prepper-plugins/opensearch/src/main/java/org/opensearch/dataprepper/plugins/source/opensearch/configuration/ConnectionConfiguration.java

```diff
@@ -14,6 +14,9 @@ public class ConnectionConfiguration {
   @JsonProperty("cert")
   private Path certPath;
 
+  @JsonProperty("certificate_content")
```

Comment:
Should we have a validation on mutual exclusivity?

```
@AssertTrue
public boolean certificateFileAndContentAreMutuallyExclusive() {
  if(certPath == null && certificateContent == null)
    return true;
  return certPath != null ^ certificateContent != null;
}
```

Reply:
Done